Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2004-09-04
Initial Package Version: 0.6.12
Upstream Status:         Not submitted 
Origin:                  http://zebra.fh-weingarten.de/~maxi/html/transcode-users/2004-08/msg00233.html
Description:             Fixes build when linking libdv

$LastChangedBy$
$Date$

diff -Naur transcode-0.6.12-orig/import/decode_dv.c transcode-0.6.12/import/decode_dv.c
--- transcode-0.6.12-orig/import/decode_dv.c	2003-12-20 11:49:13.000000000 +0000
+++ transcode-0.6.12/import/decode_dv.c	2004-09-05 01:10:40.487460896 +0000
@@ -37,8 +37,14 @@
 #include "transcode.h"
 #include "ioaux.h"
 
-#define DV_PAL_SIZE    frame_size_625_50
-#define DV_NTSC_SIZE   frame_size_525_60
+/* #define DV_PAL_SIZE    frame_size_625_50
+#define DV_NTSC_SIZE   frame_size_525_60 */
+
+static const int frame_size_625_50 = 12 * 150 * 80;
+static const int frame_size_525_60 = 10 * 150 * 80;
+#define DV_PAL_SIZE 12 * 150 * 80
+#define DV_NTSC_SIZE  10 * 150 * 8
+
 #define DV_HEADER_SIZE header_size
 
 static int verbose=TC_QUIET;
